spec/controllers/application_templates_controller_spec.rb

```diff
@@ -173,4 +173,54 @@
       end
     end
   end
+
+  describe 'POST #toggle_unavailability_enabled' do
```

Comment:
This is good, although I wouldn't mind a feature spec instead of a controller spec here.